app/jobs/digest_mail_job.rb

```diff
@@ -0,0 +1,10 @@
+class DigestMailJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(*args)
```

Comment:
This looks cool but could you add an inline comment explaining a little more about what it does? Thanks!!